test/morphometrics/test_compute_morphometrics.py

```diff
@@ -43,6 +43,12 @@ def setup(self):
         pred_myelin_path = self.test_folder_path / ('image' + str(myelin_suffix))
         self.pred_myelin = imageio_imread(pred_myelin_path, as_gray=True)
 
+        # Image to test NaN output
+        bad_pred_axon_path = self.test_folder_path / ('invalid_gratio' + str(axon_suffix))
+        self.bad_pred_axon = imageio_imread(bad_pred_axon_path, as_gray=True)
```

Comment:
Question: Should we use `imageio_imread` here or use `imread` from [`ads_utils`](https://github.com/neuropoly/axondeepseg/blob/443c0579f417f8b1d0046e3904f9d54220b6091e/AxonDeepSeg/ads_utils.py#L254)? 

Any thoughts @mathieuboudreau ?
Note that `imageio_imread` is also used in the rest of the file.

Reply:
I agree with you. This needs to be addressed in a separate PR.

Reply:
I would recommend to use `imread` from ads_utils, as it will be easier to modify or change how we read files in the entire project in the future. If features are missing from our `imread`, let me know and I can explore the best way to add them.